packet03-redirecting/Makefile

```diff
@@ -8,5 +8,6 @@ COMMON_DIR = ../common
 COPY_LOADER := xdp-loader
 COPY_STATS  := xdp_stats
 EXTRA_DEPS := $(COMMON_DIR)/parsing_helpers.h
+COMMON_OBJS += $(COMMON_DIR)/common_user_bpf_xdp.o
```

Comment:
Let's keep the assignment style consistent with the other variables (i.e., use `:=` instead of `+=`). The addition is not needed as this is set before common.mk is included below...

Reply:
thank you for your review!
changed the assignment style.